extra-android/android-file-transfer/spec

```diff
@@ -0,0 +1,4 @@
+VER=4.2
+SRCS="tbl::https://github.com/whoozle/android-file-transfer-linux/archive/v${VER}.tar.gz"
+CHKSUMS="sha256::cc607d68e8a18273c9b56975a70a0e68fbdf9d5b903b2727a345a605ff48a19f"
+SUBDIR="android-file-transfer-linux-${VER}"
```

Comment:
Please add a CHKUPDATE marker here. Please refer to https://github.com/AOSC-Dev/aosc-findupdate/blob/master/docs/config.zh-CN.md for a guide - please use Anita (https://release-monitoring.org) as version check source.

Reply:
I've mark CHKUPDATE in 7244b18, thanks ~